bip-taro-addr.mediawiki

```diff
@@ -79,7 +111,7 @@ program, as specified in BIP 341, using the included key as the internal key.
 
```

Comment:
We never use the word 'included' above; rephrase to 'using the receiver internal key as the taproot internal key' or similar? Or use a variable for that key.